cmd/fleetcontroller/main.go

```diff
@@ -3,7 +3,6 @@ package main
 import (
 	"log"
 	"net/http"
-	_ "net/http/pprof"
```

Comment:
same as above, might as well remove the whole server, or guard it with a shell variable.